### PR TITLE
Added prereq of installing VMware Tools on Node VMs

### LINKED ISSUE #4992: Update vSphere Cloud Provider prereqs to include VMware tools dependency
See the following vSphere Cloud Provider issue https://github.com/vmware/kubernetes/issues/223 and its proposed patch.

### PR DESCRIPTION
**Pull request by gaurav-nelson**
fixes #4992

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@gnufied @jhou1 Can you please review this one?

**Comment by jhou1** [comment]
LGTM

**Comment by gaurav-nelson** [comment]
@bfallonf PTAL

File: install_config/configuring_vsphere.adoc [hunk 1]
```diff
@@ -26,6 +26,13 @@ The vSphere Cloud Provider allows using vSphere managed storage within {product-
 [[vsphere-enabling]]
 == Enabling VMware vSphere Cloud Provider
 
+[IMPORTANT]
+====
+Enabling VMware vSphere requires installation of VMware Tools on each Node VM.
```
Comment (bfallonf) [inline install_config/configuring_vsphere.adoc]: @gaurav-nelson Maybe s/installation of/installing the

**Comment by bfallonf** [comment]
@gaurav-nelson One comment, but otherwise LGTM :goat:

**Comment by gaurav-nelson** [comment]
Thanks @bfallonf merging this now.

**Pull request merged**